extension/src/test/ALParser.test.ts

```diff
@@ -249,7 +249,7 @@ suite("Classes.AL Functions Tests", function () {
     }
   }
   test("API Page", function () {
-    const alObj = ALObject.getALObject(ALObjectTestLibrary.getApiPage(), true);
+    const alObj = ALParser.getALObjectFromText(ALObjectTestLibrary.getApiPage(), true);
```

Comment:
I think we should change Prettier to throw errors, we do not want this warning monsters in PRs :)

Reply:
Yeah, I think there's a bit of tweaking to be done. I didn't get most of the prettier warnings until after I created the PR and saw them in the workflow. I'm not super satisfied with the formatting at some places. I guess I could get used to parts of it but some parts just look straight up confusing :D

Reply:
And it seems as if there are still some conflicts between prettier as formatter and the built-in one...